modules/claims_api/app/models/claims_api/power_of_attorney.rb

```diff
@@ -8,6 +8,7 @@ class PowerOfAttorney < ApplicationRecord
     include FileData
     attr_encrypted(:form_data, key: Settings.db_encryption_key, marshal: true, marshaler: ClaimsApi::JsonMarshal)
     attr_encrypted(:auth_headers, key: Settings.db_encryption_key, marshal: true, marshaler: ClaimsApi::JsonMarshal)
+    attr_encrypted(:source_data, key: Settings.db_encryption_key, marshal: true, marshaler: ClaimsApi::JsonMarshal)
```

Comment:
Anyway you'd think this could be DRY-d up? `confg = {}; attr_encrypted(:attr_name, config)` etc

Reply:
Hey @kfrz i agree with you in most cases. However this dB change removed a column in favor of it being part of the encrypted hash. So a migration only commit would break implementation code. Mind you we also don’t have any production consumers using this endpoint yet so it’s safe to drop that column.